app/demo-loop/LDemoIO.cc

```diff
@@ -80,7 +80,10 @@ void from_json(const nlohmann::json& j, LDemoArgs& v)
     j.at("hepmc3_filename").get_to(v.hepmc3_filename);
     j.at("seed").get_to(v.seed);
     j.at("max_num_tracks").get_to(v.max_num_tracks);
-    j.at("max_steps").get_to(v.max_steps);
+    if (j.count("max_steps"))
```

Comment:
```suggestion
    if (j.contains("max_steps"))
```

Reply:
`contains` is a C++20 thing for standard containers so I've been using the older `count` idiom, but nljson has its own implementation separate from the standard so I'm fine with upgrading.